metadash/src/components/Dashboard.vue

```diff
@@ -1,104 +1,168 @@
+                    </v-flex>
+                  </v-layout>
+                </v-container>
+              </v-card-text>
+            </draggable>
+          </v-card>
+        </v-flex>
+        <v-flex d-flex xs12 sm6 md4>
+          <v-card flat :color="cardColor">
+            <draggable :list="medBoards" class="dragArea" row wrap @start="drag=true" :options="{group:'board'}" @end="drag=false" :move="onMoveCallback">
+              <v-flex v-for="dashboard in medBoards" :key="dashboard.name">
+                <v-card>
+                  <component v-bind:is="dashboard.module"></component>
+                </v-card>
+              </v-flex>
+              <v-card-text v-show="drag">
+                <v-container style="height: 480px;" fluid fill-height>
+                  <v-layout justify-center align-center>
+                    <v-flex text-xs-center>
+                      <v-icon x-large>add</v-icon>
+                    </v-flex>
+                  </v-layout>
+                </v-container>
+              </v-card-text>
+            </draggable>
+          </v-card>
+        </v-flex>
+      </v-layout>
+    </v-container>
+  </v-card>
 </template>
 
 <script>
 import draggable from 'vuedraggable'
 import { Plugins } from '@/plugin'
 export default {
-  name: 'hello',
+  name: 'dashboard',
   components: { draggable },
   data () {
     return {
-      actionsFab: true,
+      menu: false,
+      fab: false,
       drag: false,
-      dashboards: []
+      backImgUrl: '',
+      topBoards: {
+        'topBoard0': [],
+        'topBoard1': [],
+        'topBoard2': [],
+        'topBoard3': []
+      },
+      savedBoards: [],
+      lgBoards: [],
+      medBoards: []
     }
   },
   methods: {
     closePlugin (dashboard) {
       const index = this.dashboards.indexOf(dashboard)
       this.dashboards.splice(index, 1)
+    },
+    onMoveCallback (data, event) {
+      if (data.to.className === 'uponDragArea' && data.relatedContext.list.length > 0) {
+        return false
+      }
     }
   },
   mounted () {
+    this.$store.dispatch('getConfig', {key: 'BACKGROUND_IMG_URL'})
+      .then(data => {
+        if (data) {
+          this.backImgUrl = data.value
+        }
+      })
     for (let key in Plugins) {
       let plugin = Plugins[key]
       if (plugin.dashboard) {
         for (let subKey in plugin.dashboard) {
           let subDashboard = plugin.dashboard[subKey]
-          this.dashboards.push({'module': subDashboard.component, 'title': plugin.title, 'path': plugin.path, 'name': subDashboard.name})
+          this.savedBoards.push({'module': subDashboard.component, 'title': plugin.title, 'path': plugin.path, 'name': subDashboard.name})
         }
       }
     }
+  },
+  computed: {
+    cardColor () {
+      return this.drag ? 'grey' : 'transparent'
+    }
   }
 }
 </script>
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+.backImg {
```

Comment:
Great, css works prefecly here, just one thing, CSS class name should no use camelCase

Reply:
ok